content/blog/2022/07/v1.1.1-release.md

```diff
@@ -0,0 +1,37 @@
+---
+title: v1.1.1 is out 🎉
+authors:
+- Víctor Cuadrado Juan
+date: 2022-07-27
+---
+
+We are happy to announce the first minor release of v1.0: **v1.1.1** is
+now available!
+
+Apart from being a nice looking number, **v1.1.1** includes:
+
+* Improved the policies API for Sigstore verification by adding new
+  backwards-compatible WaPC host callback `v2/verify` functions to the API.
+  [Check them out
+    here](https://docs.kubewarden.io/writing-policies/spec/host-capabilities/signature-verifier-policies)
+  to add support for your language of choice.
+
+  This has been used in the verify-image-signatures policy to simplify
+  verification of GitHub Actions signatures and others. Read about it in [this
+  blog post](../verify-signatures-with-gha-and-prefix/).
+  
+* Made `policy-server` and `kwctl` resilient to hiccups in the upstream
+  Sigstore's repository for Fulcio and Rekor by making it optional. Now
+  Kubewarden tools will proceed forward, informing the user, and continue to
```

Comment:
Are we saying that with this change, going ahead Kubewarden tools will inform user and fail safely in case of an upstream hiccup? Or are we saying that Kubewarden tools will continue execution, inform user, and fail safely? It's unclear from this statement, which is why I'm asking.

Reply:
Reworded it in https://github.com/kubewarden/kubewarden.io/pull/119/commits/c1a1a03ae390e02ca65f9ede0eb2e6c50c60448e.

Kubewarden will do both. And Kubewarden will continue to fail safely (as it was doing before this upgrade) in case there are problems.